config/locales/en.yml

```diff
@@ -1104,12 +1104,13 @@ idv.gpo.request_another_letter.learn_more_link: Learn more about verifying your
 idv.gpo.request_another_letter.title: Request another letter?
 idv.gpo.return_to_profile: Return to your profile
 idv.gpo.title: Enter your verification code
-idv.gpo.will_send_to: 'We’ll mail a letter to:'
+idv.gpo.will_send_to: 'You’ll get a letter in the mail from %{app_name} in 5 to 10 days at the address we verified and associated with you:'
 idv.images.come_back_later: Letter with a check mark
 idv.messages.activated_html: Your identity has been verified. If you need to change your verified information, please %{link_html}.
 idv.messages.activated_link: contact us
-idv.messages.come_back_later_html: 'Letters take <strong>5 to 10 days</strong> to arrive. Sign back in to enter your verification code once you get your letter.'
+idv.messages.come_back_later: Sign back in and enter the verification code when your letter arrives.
 idv.messages.confirm: We secured your verified information
+idv.messages.contact_sp_html: Contact&nbsp;%{sp_link} if you need to access their services before your letter arrives.
```

Comment:
OK, so: the i18n linter yells at me because `contact_sp_html` is an `_html` string but doesn't contain any HTML. `%{sp_link}` is a hyperlink we interpolate. Taking the `_html` out causes raw HTML to be displayed.

I have put an otherwise-needless `&nbsp;` here to make things happy, but this is obviously a very gross approach. I am out tomorrow (Wednesday) if anyone wants to apply an obvious correction; otherwise, I'll pick this up Thursday morning.

Reply:
I think you can suffix the interpolated key as `%{sp_link_html}` for it to understand the HTML is in the interpolated value.

https://github.com/18F/identity-idp/blob/b77c611670c706e1adf41f6ae42d0ea21e83f960/spec/i18n_spec.rb#L387-L391

Reply:
Ah-ha! Thanks for this pointer.

Since it still took me a little bit of trial-and-error, a pointer for the next person:
- The translation key should still keep the `_html` suffix
- Including an interpolation ending in `_html` causes the linter to not object to the translation not directly including HTML